generation/update_versions.sh

```diff
@@ -78,19 +78,21 @@ for path in $(find . -mindepth 2 -maxdepth 2 -name pom.xml | sort | xargs dirnam
       # maven_latest_version stores Major.Minor.Patch or the entire version
       # maven_latest_trailing stores alpha/beta/etc. or nothing
       maven_metadata_version=$(echo "${metadata_file}" | grep 'latest' | cut -d '>' -f 2 | cut -d '<' -f 1)
-      maven_latest_version=$(echo "${maven_metadata_version}"  | cut -d "-" -f1)
-      maven_latest_trailing=$(echo "${maven_metadata_version}"  | cut -s -d "-" -f2-)
-
-      major_version=$(echo "${maven_latest_version}" | cut -d "." -f1)
-      minor_version=$(echo "${maven_latest_version}" | cut -d "." -f2)
-      patch_version=$(echo "${maven_latest_version}" | cut -d "." -f3)
-      patch_version_bump=$((patch_version + 1))
-      if [[ -z "${maven_latest_trailing}" ]]; then
-        maven_version_bump="${major_version}.${minor_version}.${patch_version_bump}"
-      else
-        maven_version_bump="${major_version}.${minor_version}.${patch_version_bump}-${maven_latest_trailing}"
-      fi
-      new_version="${artifactId}:${maven_metadata_version}:${maven_version_bump}-SNAPSHOT"
+
+      # TODO: Confirm if release-please is able to bump snapshot
+#      maven_latest_version=$(echo "${maven_metadata_version}"  | cut -d "-" -f1)
```

Comment:
I think you can just delete this code. We have version history for the event of having to restore.